### PR TITLE
Feature: Add Notes to Person

### PR DESCRIPTION
**Pull request by LOLIPOP-INTELLIGENCE**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ListCommand.java
Comment (sooyitao) [inline src/main/java/seedu/address/logic/commands/ListCommand.java:1]: I think more list command messages can be added to help users. Currently, none of the list inputs displays MESSAGE_USAGE. Maybe you can add a message for Listed all by name/email and display MESSAGE_USAGE when invalid command format is used.

File: src/main/resources/view/PersonListCard.fxml [hunk 1]
```diff
@@ -31,6 +31,7 @@
       <Label fx:id="phone" styleClass="cell_small_label" text="\$phone" />
       <Label fx:id="address" styleClass="cell_small_label" text="\$address" />
       <Label fx:id="email" styleClass="cell_small_label" text="\$email" />
+      <Label fx:id="notes" styleClass="cell_small_label" text="\$notes" />
     </VBox>
   </GridPane>
-</HBox>
+</HBox>
```
Comment (sooyitao) [inline src/main/resources/view/PersonListCard.fxml:37]: Add newline at EOF to pass CheckStyle.

**Review by sooyitao — APPROVED** [review]
The check can be easily fixed by adding newline at EOF of PersonListCard.fxml.

**Review by ty4g1 — COMMENTED** [review]
A few suggestions for next milestone.
Avoid displaying the notes in the UI, I think with longer notes it may appear cluttered. Perhaps we can use an icon to indicate a certain contact has notes attached.
There can be a separate command that takes in a contact name or index, and displays the notes in the output window.

**Pull request merged**